osu.Game/Overlays/Chat/ChannelControl/ControlItemClose.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osuTK;
+
+namespace osu.Game.Overlays.Chat.ChannelControl
+{
+    public class ControlItemClose : OsuClickableContainer
+    {
+        private readonly SpriteIcon icon;
+
+        [Resolved]
+        private OsuColour osuColour { get; set; } = null!;
+
+        public ControlItemClose()
+        {
+            Alpha = 0f;
+            Size = new Vector2(20);
+            Child = icon = new SpriteIcon
+            {
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Scale = new Vector2(0.75f),
+                Icon = FontAwesome.Solid.TimesCircle,
+                RelativeSizeAxes = Axes.Both,
+            };
+        }
+
+        protected override bool OnMouseDown(MouseDownEvent e)
+        {
+            icon.ScaleTo(0.5f, 1000, Easing.OutQuint);
+            return base.OnMouseDown(e);
+        }
+
+        protected override void OnMouseUp(MouseUpEvent e)
+        {
+            icon.ScaleTo(0.75f, 1000, Easing.OutElastic);
+            base.OnMouseUp(e);
+        }
+
```

Comment:
This should probably be using the newer transform logic (see https://github.com/ppy/osu/blob/18c144395ae7644601b09da7a112d963a0bcb27c/osu.Game/Graphics/UserInterface/OsuAnimatedButton.cs#L98-L126). The scale is too harsh (looks to have been copied from the old tab implementation).